### ISSUE TITLE
Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by g-s-2018** [vbsreenu/Sam2Consensus#1]
Dear, 

Does this tool is suitable for working with long reads direct RNA seq? I am getting this error msg 

Segmentation fault (core dumped)

Thanks in advance,

**Comment by vbsreenu** [comment]
No, I have not tested this tool with long reads. In the code, I have set the read length limit to 1024 bp, so it gives you segmentation fault if the length in longer.

**Comment by g-s-2018** [comment]
Thanks for your replay, 

My longest read size is 6612 bp. Can I edit this part in the script?

Many thanks
